Tests/OpenFeatureTests/Helpers/DoSomethingProvider.swift

```diff
@@ -65,4 +65,8 @@ class DoSomethingProvider: FeatureProvider {
     public struct DoMetadata: ProviderMetadata {
         public var name: String? = DoSomethingProvider.name
     }
+
+    func initWithError() {
```

Comment:
I believe instead of introducing this here we should use `AlwaysBrokenProvider` in the test.